readme.txt

```diff
@@ -224,9 +224,11 @@ Remember to always make a backup of your database and files before updating!
 
 * Tweak - Minify the Freemius svg assets. [TEC-3215]
 * Tweak - Remove "(beta)" label from URL source type of import. [TEC-3289]
+* Tweak - Reword the Filter Activation setting's labels for better clarity. Add filters `tribe_events_liveupdate_automatic_label_text` and `tribe_events_liveupdate_manual_label_text` to allow Filter Bar to include itself in those lables. [FBAR-79]
```

Comment:
Please, break this changelog entry in two as they would target 2 diff. audiences; one for the UI update, one for developers.